src/controllers/dynakube/dynakube_controller.go

```diff
@@ -316,7 +316,7 @@ func (controller *DynakubeController) reconcileActiveGate(ctx context.Context, d
 
 func (controller *DynakubeController) reconcileActiveGateProxySecret(ctx context.Context, dynakubeState *status.DynakubeState) bool {
```

Comment:
No unit test was changed when this methods logic changed. Please edit or add an unit test accordingly